### PR TITLE
Remove usage of RuntimeDetector

### PR DESCRIPTION
**Pull request by spacey-sooty**

### CODE REVIEW

File: photon-core/src/main/java/org/photonvision/vision/camera/USBCameras/USBCameraSource.java [hunk 1]
```diff
@@ -113,7 +112,7 @@ protected GenericUSBCameraSettables createSettables(
         GenericUSBCameraSettables settables;
 
         if (quirks.hasQuirk(CameraQuirk.LifeCamControls)) {
-            if (RuntimeDetector.isWindows()) {
+            if (System.getProperty("os.name").startsWith("Windows")) {
```
Comment (mcm001) [inline photon-core/src/main/java/org/photonvision/vision/camera/USBCameras/USBCameraSource.java:115]: This feels hacky. Let's keep some form of RuntimeDetector around.

Comment (spacey-sooty) [reply photon-core/src/main/java/org/photonvision/vision/camera/USBCameras/USBCameraSource.java:115]: This is what RuntimeDetector did internally. We could add this functionality to platform?

Comment (Juniormunk) [reply photon-core/src/main/java/org/photonvision/vision/camera/USBCameras/USBCameraSource.java:115]: This should be something like Platform getcurrentplatform ostype = windows right?

**Review by stephenjust — APPROVED** [review]
Works fine on Orange Pi 5 Plus - Note the merge conflict in PhotonTargetingJniLoader, the platform check for athena is removed.

**Review by mcm001 — APPROVED** [review]

**Pull request merged**